lib/ssh_scan/public_key.rb

```diff
@@ -39,10 +39,11 @@ def fingerprint_md5
 
       def fingerprint_sha1
         SSHKey.sha1_fingerprint(@key_string)
-      end    
+      end
 
       def fingerprint_sha256
-        SSHKey.sha256_fingerprint(@key_string)
+        # We're translating this to hex because the SSHKEY default isn't as useful for comparing with SSHFP records
+        Base64.decode64(SSHKey.sha256_fingerprint(@key_string)).hexify(:delim => ":")
```

Comment:
I can see why sshkey base64'd this before, it's simply a shorter format to express the same data.  I think hex is just easier to see if you're manually reviewing.  We can then write rules to catch weird violations.